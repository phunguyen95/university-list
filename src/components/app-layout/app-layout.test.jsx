import { render } from "@testing-library/react";
import AppLayout from "./app-layout";
import { Router } from "react-router-dom";
import { createMemoryHistory } from "history";

test("renders applayout  view without crashing", () => {
    const history = createMemoryHistory();

  const { asFragment } = render(<Router history={history}><AppLayout>
      <div>Test</div>
    </AppLayout></Router>);

  expect(asFragment()).toMatchSnapshot();
});
