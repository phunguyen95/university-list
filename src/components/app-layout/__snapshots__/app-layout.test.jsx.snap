// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders applayout  view without crashing 1`] = `
<DocumentFragment>
  <div
    class="rootLayout"
  >
    <div
      class="backgroundImage"
    />
    <div
      class="layout"
    >
      <div>
        Test
      </div>
    </div>
  </div>
</DocumentFragment>
`;
